lib/src/actions/user/userproviders/freebsd.rs

```diff
@@ -0,0 +1,55 @@
+use super::UserProvider;
```

Comment:
nit: can we call this directory just `providers`?

Reply:
We can. I just just did UserProvider to make it more distinct in name from the PackageProvider